src/crudeditor-lib/views/create/workerSagas/save.js

```diff
@@ -125,9 +125,15 @@ export default function*({
 
   if (afterAction === AFTER_ACTION_NEW) {
     // create another instance
+    const { defaultNewInstance } = modelDefinition.ui.create;
+
     yield put({
       type: VIEW_INITIALIZE,
-      payload: { predefinedFields: {} },
+      payload: {
+        predefinedFields: {
+          ...(defaultNewInstance ? defaultNewInstance() : null)
```

Comment:
According to [documentation](https://github.com/OpusCapita/react-crudeditor/blob/master/README.md#definition-object-structure) **defaultNewInstance()** must be called with Search View state as an argument